src/uk/gov/hmcts/contino/YarnBuilder.groovy

```diff
@@ -98,6 +112,7 @@ class YarnBuilder extends AbstractBuilder {
     finally {
       steps.junit allowEmptyResults: true, testResults: 'functional-output/**/*result.xml'
       steps.archiveArtifacts allowEmptyArchive: true, artifacts: 'functional-output/**'
+      savePodLogs("full-functional")
```

Comment:
i'm not sure if it's a safe assumption that you will be logged into the cluster context for this.

Reply:
This file is no longer subject to change in this PR.
Marking this as resolved.